app/Http/Requests/Worker/Supervisor/AddWorkerToShiftRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Requests\Worker\Supervisor;
+
+use App\Models\Shift;
+use App\Models\User;
+use App\Models\Worker;
+use Illuminate\Foundation\Http\FormRequest;
+
+class AddWorkerToShiftRequest extends FormRequest {
+	protected $shift;
+	protected $workplace;
+	/**
+	 * Determine if the user is authorized to make this request.
+	 *
+	 * @return bool
+	 */
+	public function authorize() {
+		$this->shift = $this->route('shift');
+		$this->workplace = $this->route('workplace');
```

Comment:
change the column name to something else - worker makes more sense than name (for example)